pkg/vault/contracts/ProtocolFeesCollector.sol

```diff
@@ -86,11 +90,26 @@ contract ProtocolFeesCollector is IProtocolFeesCollector, Authentication, Reentr
         emit FlashLoanFeePercentageChanged(newFlashLoanFeePercentage);
     }
 
+    function setFlashLoanFeeDiscount(address[] memory users, bool[] memory isDiscounted)
```

Comment:
make isDiscounted not an array, it is unreal situation when setup mixed values

Reply:
Done